FAQ.md

```diff
@@ -599,7 +695,7 @@ TiDB 读流量可以通过增加 TiDB server 进行扩展，总读容量无限
 - KV entry 的总条数不超过 30w
 - KV entry 的总大小不超过 100MB
 
-在 Google 的 Cloud Spanner 上面，也有类似的[限制](https://cloud.google.com/spanner/docs/limits)。
+在 Google 的 Cloud Spanner 上面，也有类似的 [限制](https://cloud.google.com/spanner/docs/limits)。
```

Comment:
**Please delete all the similar whitespace before linked Chinese words involved in this PR.**